core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -200,118 +200,6 @@ public void testScenario_111() throws Exception {
    }
 
    /**
-    * ISPN-3315: the remote get is trigger in stable state and the reply received after the rebalance_start command.
-    * However, the old owner will receive the request after the state transfer and he no longer has the key.
```

Comment:
Is there a reason for topology being installed on one node later than on the others?